pinot-clients/pinot-java-client/pom.xml

```diff
@@ -61,4 +61,13 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <profiles>
+    <profile>
+      <id>pinot-fastdev</id>
+      <properties>
+        <shade.phase.prop>none</shade.phase.prop>
+      </properties>
+    </profile>
+  </profiles>
```

Comment:
This is basically what have been added in all `pom.xml` that changes the default value of `shade.phase.prop`, which is `none` by default (in the root pom).

In this case (although it is not seen in the PR by default) the value is changed to package in line 34. This is one of the three methods to change the property we are using. I've tried to unify them as well, but I don't want to make this PR more complex.

Reply:
Why do we need all of these overrides in the child POMs if we're already setting the property value for `shade.phase.prop` to `none` in the `pinot-fastdev` profile definition in the root POM?

Reply:
that doesn't seem to work. It looks like properties defined in this pom have higher priority than properties defined in a profile defined in the parent of this pom.

Reply:
Maybe the diff tricks you, but this (and other) pom.xml set `shade.phase.prop` to true in line 34